yt_dlp/extractor/crunchyroll.py

```diff
@@ -700,7 +700,7 @@ def _real_extract(self, url):
 
 class CrunchyrollBetaIE(CrunchyrollBaseIE):
     IE_NAME = 'crunchyroll:beta'
-    _VALID_URL = r'https?://beta\.crunchyroll\.com/(?P<lang>(?:\w{1,2}/)?)watch/(?P<internal_id>\w+)/(?P<id>[\w\-]+)/?(?:\?|$)'
+    _VALID_URL = r'https?://beta\.crunchyroll\.com/(?P<lang>(?:\w{1,2}/)?)watch/(?P<internal_id>\w+)/(?P<id>[\w\-]*)/?(?:\?|$)'
```

Comment:
better to rename the groups as `id` and `display_id`

Reply:
Yeah, that should get changed, but it seems like it fits better as part of my upcoming beta playlist support PR. Mind if I put it off till then?

Reply:
Better to handle the whole thing in that PR then imho. Due to the huge scope of youtube-dl, if we make a PR of each small change, we'll never get anywhere. Feel free to make multiple changes in one PR as long as they concern the same site/extractor

Reply:
Alright. Will do.